### PR TITLE
docs(network): introduce network docs

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: docs/network.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+
+> **NOTE** As of playwright v0.13.0, Playwright is not yet capable of tracking websocket messages.
+
+
+## Monitor all network activity in page
+
+```js
+const page = await browser.newPage();
+page.on('request', request => console.log('>>', request.method(), request.url()));
```
Comment (yury-s) [inline docs/network.md:17]: nit: how about using the fancy symbols that we use internally (◀  and ►) ?

File: docs/network.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+page.on('response', response => console.log('<<', response.status(), response.url()));
+await page.goto('https://example.com');
+```
+
+#### API reference
```
Comment (yury-s) [inline docs/network.md:22]: I like the back-references to the API in each section, but it somewhat hard to distinguish between the main examples and the following refs. Maybe we can tweak the formatting a bit (smaller font for the API links or bigger margin before the next section) ?

**Review by pavelfeldman — APPROVED** [review]

**Review by yury-s — APPROVED** [review]

File: docs/network.md (hunk 1 above)
Comment (pavelfeldman) [reply docs/network.md:17]: Yury, I'll merge it in and we can patch it in the tree!

File: docs/network.md (hunk 2 above)
Comment (yury-s) [reply docs/network.md:22]: Adding  
```html
<br/> <br/>
```
seems to work well in input.md, let's do the same here.

**Pull request merged**